data/README.rst

```diff
@@ -0,0 +1,107 @@
+Software Directory README
```

Comment:
The ``rst`` extension of the README file is pretty uncommon. If we use a more common extension, more clients will automagically render the content properly. I'd suggest using no extension (just ``README``) or the markdown extension (``README.md``)